eZ/Publish/Core/Search/Common/IndexerErrorHandler.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace eZ\Publish\Core\Search\Common;
+
+use eZ\Publish\SPI\Persistence\Content\ContentInfo;
+
+/**
+ * Interface for handling errors during indexing.
+ */
+interface IndexerErrorHandler
```

Comment:
Missing hasErrors and getErrors

Reply:
Added in https://github.com/ezsystems/ezpublish-kernel/pull/2221/commits/4f8d3f0543ea1ddaae33c42a945ab2cc63c2def2.